README.md

```diff
@@ -1 +1,285 @@
+  - [Backwards compatibility in `0.0.z` and `0.y.z` version](#backwards-compatibility-in-00z-and-0yz-version)
+- [About Mineiros](#about-mineiros)
+- [Reporting Issues](#reporting-issues)
+- [Contributing](#contributing)
+- [Makefile Targets](#makefile-targets)
+- [License](#license)
+
+## Module Features
+
+In contrast to the plain `terraform_google_bigquery_dataset_iam_binding`, `terraform_google_bigquery_dataset_iam_member` resources this module has better features.
+While all security features can be disabled as needed best practices
+are pre-configured.
+
+<!--
+These are some of our custom features:
+
+- **Default Security Settings**:
+  secure by default by setting security to `true`, additional security can be added by setting some feature to `enabled`
+
+- **Standard Module Features**:
+  Cool Feature of the main resource, tags
+
+- **Extended Module Features**:
+  Awesome Extended Feature of an additional related resource,
+  and another Cool Feature
+
+- **Additional Features**:
+  a Cool Feature that is not actually a resource but a cool set up from us
+
+- _Features not yet implemented_:
+  Standard Features missing,
+  Extended Features planned,
+  Additional Features planned
+-->
+
+## Getting Started
+
+Most basic usage just setting required arguments:
+
+```hcl
+module "terraform-google-bigquery-dataset-iam" {
+  source = "github.com/mineiros-io/terraform-google-bigquery-dataset-iam.git?ref=v0.1.0"
+}
+```
+
+## Module Argument Reference
+
+See [variables.tf] and [examples/] for details and use-cases.
+
+### Top-level Arguments
+
+#### Module Configuration
+
+- **`module_enabled`**: _(Optional `bool`)_
+
+  Specifies whether resources in the module will be created.
+  Default is `true`.
+
+- **`module_depends_on`**: _(Optional `list(dependencies)`)_
+
+  A list of dependencies. Any object can be _assigned_ to this list to define a hidden external dependency.
+
+  Example:
+  ```hcl
+  module_depends_on = [
+    google_network.network
+  ]
+  ```
+
+#### Main Resource Configuration
+
+- **`dataset_id`**: **_(Required `string`)_**
+
+  A unique ID for this dataset, without the project name. The ID must contain only letters `(a-z, A-Z)`, numbers `(0-9)`, or underscores `(_)`. The maximum length is 1,024 characters.
+
+- **`role`**: **_(Required `list(role)`)_**
+
+  The role that should be applied. Only one `'google_secret_manager_secret_iam_binding'` can be used per role. Note that custom roles must be of the format `'[projects|organizations]/{parent-name}/roles/{role-name}'`.
+
+- **`project`**: _(Optional `string`)_
+
+  The ID of the project in which the resource belongs. If it is not provided, the provider project is used.
+  Default is `null`.
+
+- **`members`**: _(Optional `set(string)`)_
+
+  Identities that will be added/set to/for the role. Each entry can have one of the following values: `'allUsers'`, `'allAuthenticatedUsers'`, `'serviceAccount:{emailid}'`, `'group:{emailid}'`, `'domain:{domain}'`.
```

Comment:
please make this a list and use double quotes as terraform does not support single quotes as values